apps/jobs/models.py

```diff
@@ -70,9 +76,9 @@ class Submission(TimeStampedModel):
     started_at = models.DateTimeField(null=True, blank=True)
     completed_at = models.DateTimeField(null=True, blank=True)
     when_made_public = models.DateTimeField(null=True, blank=True)
-    input_file = models.FileField(upload_to=RandomFileName("submission_files/submission"))
-    stdout_file = models.FileField(upload_to=RandomFileName("submission_files/submission"), null=True, blank=True)
-    stderr_file = models.FileField(upload_to=RandomFileName("submission_files/submission"), null=True, blank=True)
+    input_file = models.FileField(upload_to=get_upload_path)
```

Comment:
@TroJan I think if we try to add a hook in `RandomFileName` then we may not be able to use it to other places, like we have used it[ here](https://github.com/Cloud-CV/EvalAI/blob/master/apps/challenges/models.py#L35). What are your views about it?